OrrRock/OrrRock/ViewControllers/OnBoardingViewController/OnBoardingViewController.swift

```diff
@@ -152,9 +67,89 @@ class OnBoardingViewController: UIViewController , UISheetPresentationController
         }
     }
     
+    func setDelegate(){
+        self.delegate = self
+        self.dataSource = self
+        self.navigationController?.isNavigationBarHidden = true
+        if let firstVC = vcArray.first {
+            setViewControllers([firstVC], direction: .forward, animated: false, completion: nil)
+        }
+    }
+    
     @objc
     private func pressNextButton(_ sender: UIButton) {
-        UserDefaults.standard.set(true, forKey: "watchOnBoard")
-        self.dismiss(animated: true)
+        self.goToNextPage()
+        if pageControl.currentPage == 2 {
+            self.navigationController?.isNavigationBarHidden = false
+            self.navigationController?.popToRootViewController(animated: false)
+            UserDefaults.standard.set(true, forKey: "watchOnBoard")
+            
+        }
+        pageControl.currentPage += 1
+        //이동
+    }
+    
+}
+
+extension OnBoardingViewController : UIPageViewControllerDataSource{
```

Comment:
```suggestion
extension OnBoardingViewController: UIPageViewControllerDataSource {
```

p1;

코드 컨벤션 지켜주세요!